image_restoration/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.5)
+project(image_restoration)
+
+find_package(catkin REQUIRED COMPONENTS
+  rospy
+  sensor_msgs
+  cv_bridge
+  catkin_virtualenv
+)
+
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/weights)
+set(MODEL_FILE ${CMAKE_CURRENT_SOURCE_DIR}/weights/RealESRGAN_x4plus.pth)
+
+if(NOT EXISTS ${MODEL_FILE})
```

Comment:
Please use `${WGET_EXECUTABLE}` and write the downloading procedure as a target. Something like

```cmake
add_custom_target(create_weights_dir
    COMMAND ${CMAKE_COMMAND} -E make_directory "${PROJECT_SOURCE_DIR}/weights"
)
add_custom_target(RealESRGAN_x4plus.pth ALL
    DEPENDS create_weights_dir
    COMMAND ${WGET_EXECUTABLE} -q "https://github.com/xinntao/Real-ESRGAN/releases/download/v0.1.0/RealESRGAN_x4plus.pth" -O "${PROJECT_SOURCE_DIR}/weights/RealESRGAN_x4plus.pth"
)

```